README.md

```diff
@@ -9,6 +9,12 @@
 
 oc-new-app is a Github Action for deploying and exposing an application on Openshift.
 
+## Prerequisites
```

Comment:
should an openshift cluster be a prereq?

Reply:
It is, let me update that too.